EXAMPLES.md

```diff
@@ -76,40 +132,96 @@ named ```john```
 Output example :
 
 ```json
+{
+    "id": "ERUY93RPJHWGUBF0IDQ8",
+    "value": "5Of4usgFeLVH5Qr0NU6UxW0nyeYzdNQaJKQS=dRt",
+    "createDate": "2019-12-26T10:50:02Z",
+    "lastUsedDate": "2019-12-26T10:50:02Z",
+    "status": "Active",
+    "userId": "235437388852"
+}
+```
+
+### Inject accesskey and secretkey
+
+```sh
+vaultclient generate-account-access-key --name john \
+--accesskey=AKIA5X47K766FWESB7PL \
+--secretkey=+870aSdHxh9PPS7jxcRuYIMeX9P0ytDYVd/Q8s1B
+```
+
+The previous example will generate inject the supplied account access key
+and secret key for the account named ```john```
+
+Output example :
 
+```json
 {
-    "id": "XMHR9IQ9UYN56W1OSN2S",
-    "value": "5tK4XOid7pXss66A7Jn=Yz7ybnMIB4Uf/BjavN58",
-    "createDate": "2016-07-02T21:47:57Z",
-    "lastUsedDate": "2016-07-02T21:47:57Z",
+    "id": "AKIA5X47K766FWESB7PL",
+    "value": "+870aSdHxh9PPS7jxcRuYIMeX9P0ytDYVd/Q8s1B",
+    "createDate": "2019-12-26T10:50:52Z",
+    "lastUsedDate": "2019-12-26T10:50:52Z",
     "status": "Active",
+    "userId": "235437388852"
 }
+```
 
+### Update Account Quota
```

Comment:
Previous title lines have pretty consistently been set in "down" style (initial capital only), so I left them alone (I prefer the "up" style, with each word capitalized, that you have here). Be consistent, whichever way you decide to go.

Reply:
Understood and changed :)